example-config.yaml

```diff
@@ -0,0 +1,10 @@
+chain_config: test-chain/genesis.json
+data_dir: test-chain
+jsonrpc_addr: "0.0.0.0:10001"
+grpc_addr: "0.0.0.0:20001"
+seal: true
+network:
+  libp2p_addr: "127.0.0.1:1478"
+log_to: node.log
+
+# for additional parameters check out the docs page: https://edge-docs.polygon.technology/docs/configuration/sample-config
```

Comment:
This configuration should not be on the repo but on the docs

Reply:
Removed example config file and added command to create default config yaml file on the fly with default configuration.   
By running `server export` the user gets new `default-config.yaml` that can be used to start the server .   
1ef09e369a8f08a202d3641e7402f04762cb4384

Also, took care of some default server command values, as they were hardcoded instead of them being pulled from default configuration.